src/main/java/frc/robot/Constants.java

```diff
@@ -11,70 +16,77 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-    public static final int primaryController = 0;
+  public static final class CanId{
+    public static final int climberLiftLead = 1;
+    public static final int climberLiftFollow = 2;
+    public static final int climberRotate = 3;
+    public static final int intakeLower = 5;
+    public static final int upperConveyor = 4;
+    public static final int shooterLead = 6;
+    public static final int shooterFollow = 7;
+    public static final int leftDriveLead = 13;
+    public static final int leftDriveFollow = 12;
+    public static final int rightDriveLead = 11;
+    public static final int rightDriveFollow = 10;
+    public static final int compressor = 8;
+    public static final int solenoidPort = 0;
+  }
 
-    public final class CanId{
-        public static final int climberLiftLead = 1;
-        public static final int climberLiftFollow = 2;
-        public static final int climberRotate = 3;
-        public static final int intakeLower = 5;
-        public static final int upperConveyor = 4;
-        public static final int shooterLead = 6;
-        public static final int shooterFollow = 7;
-        public static final int leftDriveLead = 13;
-        public static final int leftDriveFollow = 12;
-        public static final int rightDriveLead = 11;
-        public static final int rightDriveFollow = 10;
-        public static final int compressor = 8;
-        public static final int solenoidPort = 0;
+  public static final class Drivetrain{
+    public static final class Feedforward{
+      //Feedforwards from sysid
+      public final class Left{
+        public static final double kS = 0.56131;
+        public static final double kV = 2.065;
+        public static final double kA = 0.37539;
+      }
+      public final class Right{
+        public static final double kS = 0.55809;
+        public static final double kV = 2.0644;
+        public static final double kA = 0.19512;
+      }
     }
 
-    public final class Drive{
-        public final class Feedforward{
-            //Feedforwards from sysid
-            public final class Left{
-                public static final double kS = 0.56131;
-                public static final double kV = 2.065;
-                public static final double kA = 0.37539;
-            }
-            public final class Right{
-                public static final double kS = 0.55809;
-                public static final double kV = 2.0644;
-                public static final double kA = 0.19512;
-            }
-        }
-        public static final double kTrackWidth = 1; // m/s
-        public static final boolean kInvertDrive = false;
+    public static final class Dimensions{
+      public static final double wheelCircumferenceMeters = Units.inchesToMeters(6*Math.PI);
+      //TODO Measure Trackwidth
+      public static final double trackWidthMeters = Units.inchesToMeters(30);
```

Comment:
Why this hardcoded `30` here?

Reply:
It's the empirically measured trackwidth in inches which has not been measured theoretically or empirically yet :man_shrugging: